sampledb/reader.xsh

```diff
@@ -0,0 +1,92 @@
+import pandas as pd
+from time import time
+from sampledb.sampledatabase import SampleDatabase
+
+
+def get_uid_from_qr():
+    a = $(zbarcam)
+    b = a.split()
+    c = [z.strip('QR-Code:') for z in b]
+    d = set(c)
+    return d
+
+def write_sample_spreadsheet(uids, sdb):
+    if len(uids) == 0:
+        return
+
+    sdb.load_schema('sample_schema.json')
```

Comment:
We might want a dedicated order to the fields in excel, at least for consistency.